jtune.py

```diff
@@ -1777,6 +1780,20 @@ def get_jmap_data(pid=None, procdetails=None):
 # Main
 user = getpass.getuser()
 
+
```

Comment:
@ericbullen  I think I got it all, tested in rotation on/off, also added a fallback if no rotated style log was found.